nextcord/enums.py

```diff
@@ -192,6 +192,7 @@ class ChannelType(Enum):
     public_thread = 11
     private_thread = 12
     stage_voice = 13
+    forum = 15
```

Comment:
ChannelType in api.rst should be updated

As well as other public classes added by this pr